docs/tutorials/uniq-factories/uniq-offer/uniq-offer.experimental.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Uniq Offer'
+
+order: 1
+---
+
+# Uniq Offer
+
+## Overview of Uniq Offer feature
+
+Uniq Offer allow user to make a buy offer directly to the owner of Uniq or Uniq Factory if it's tradable.
+
+Once these offers are made, owner of Uniq or Uniq Factory can accept these offers as long as offer is still valid. This means offers remain valid even if the Uniq changes hands, ensuring your offer stands until it expires or is withdrawn.
+
+Upon making an offer, `UOS` is securely locked through smart contracts, safeguarding Uniq owners against fake bids.
+
+You can cancel your offer anytime or let it expire. Once an offer expires, anyone can cancel the offer. When an offer is canceled, the offered price will be transferred back to the user who made the offer.
+
+When an offer is accepted, the Uniq transfers to the specified buyer address, ensuring a smooth transition of ownership. The fund will also be released and transferred to the owner of the Uniq. If the promoter option is configured, fund will be distributed accordingly to [2nd Hand Sale Policy](../../../general/antelope-ultra/2nd-hand-sale.md)
```

Comment:
Should be
`When an offer is accepted, the Uniq will be transferred to the buyer (or receiver if specified when making an offer), ...`

Reply:
fixed text and update link